src/client/src/BdmsAuthProvider.js

```diff
@@ -27,23 +27,44 @@ const AuthenticationStoreSync = () => {
 };
 
 export const BdmsAuthProvider = props => {
-  const onSigninCallback = user => {
-    window.history.replaceState({}, document.title, window.location.pathname);
-  };
-
-  const oidcConfig = {
-    authority: window.env.AUTHORITY,
-    client_id: window.env.CLIENT_ID,
-    scope: "openid profile email",
-    redirect_uri: window.location.origin + window.location.pathname,
-    userStore: new WebStorageStateStore({ store: window.localStorage }),
-    onSigninCallback,
-  };
-
-  return (
+  const [serverConfig, setServerConfig] = useState(undefined);
+  const [oidcConfig, setOidcConfig] = useState(undefined);
+
+  useEffect(() => {
+    fetch("/api/v2/settings/auth")
+      .then(res =>
+        res.ok
+          ? res.json()
+          : Promise.reject(Error("Failed to get auth settings from API")),
+      )
+      .then(setServerConfig)
+      .catch(setServerConfig(undefined));
+  }, []);
+
+  useEffect(() => {
+    if (!serverConfig) return;
+
+    const onSigninCallback = user => {
+      // remove response code form url.
+      window.history.replaceState({}, document.title, window.location.pathname);
+    };
+
+    setOidcConfig({
+      authority: serverConfig.authority,
+      client_id: serverConfig.audience,
+      scope: "openid profile email",
```

Comment:
PP:
Ich hätte jetzt die _scopes_ auch noch in die Api verschoben, Einfach damit sämtliche OIDC Konfiguration an einem Ort ist.